velox/common/memory/tests/AllocationTest.cpp

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <gtest/gtest.h>
+
+#include "velox/common/memory/Memory.h"
+
+using namespace ::testing;
+using namespace facebook::velox::memory;
+
+class AllocationTest : public testing::Test {};
+
+TEST_F(AllocationTest, pageRunMerge) {
+  Allocation allocation;
+  const uint64_t kFakeAddrValue = 4096;
```

Comment:
kFakeAddr1, kFakeAddr2, kFakeAddr3 are a bit hard to understand. Do we need to include 'Fake' in the names of variables? Can we come up with more descriptive names? Perhaps, separate each test case with an empty line for readability. kXxx naming applies only to static constants. Please, rename or change to static.